package/iframe/js/main.js

```diff
@@ -0,0 +1,150 @@
+var activities = require('./activities');
+var features = require('./features');
+var logger = require('./logger');
+var messages = require('./messages');
+var translations = require('./translations');
+
+
+logger.log('MKT_URL:', process.env.MKT_URL);
+
+function buildQS(profile) {
+    var qs = [];
+
+    try {
+        // navigator.mozMobileConnections is the new API.
+        // navigator.mozMobileConnection is the legacy API.
```

Comment:
If we care about Firefox OS 1.1, then yes. Since it was there I don't think it's worth removing.